### PR TITLE
Run event emitters on nextTick.

### LINKED ISSUE #3: virtual-serialport shouldn't have synchronous emitters
I was trying to use the virtual-serialport package to mock some code that worked fine with node-serialport and actual hardware. However, virtual-serialport makes synchronous emit() calls, meaning that instead of being able to assign callbacks/event handlers after open/write/etc, callbacks execute during the call (.write() calls emit("data") which calls all callbacks right then). Emits should be wrapped in something like setImmediate in order to simulate expected usage.

### PR DESCRIPTION
**Pull request by qdot**
Fixes #3. 

Should probably also look at doing this for callbacks, which should also fire after the enclosing function has run to completion, but this at least fixes the issues I was having.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ var VirtualSerialPort = function(path, options, openImmediately, callback) {
     };
 
     if(openImmediately || openImmediately === undefined || openImmediately === null) {
-        process.nextTick(function() {
+        process.nextTick(() => {
```
Comment (ecksun) [inline index.js]: I believe this library still works with node 0.10 (which I personally require at the moment). However this change requires at least 4. You should probably not introduce a dependency on a new node version in a small bugfix like this.

Comment (qdot) [inline index.js]: Whoops! Thanks for pointing that out. Force of habit.

Comment (ecksun) [inline index.js]: should you not fix it then? :)

Comment (reyiyo) [inline index.js]: Hi @qdot!

Thank you very much for yout PR! Do you think you can fix this so we can merge and release it?

**Comment by qdot** [comment]
Ok, I think that fixes it.

**Pull request merged**